docs/corpus.markdown

```diff
@@ -2,44 +2,45 @@
+One file that each corpus should generate is a CSV file in the `data` directory named `metadata.csv`. This file should contain any additional information about individual documents that may be relevant. For example, the `metadata.csv` generated by the Federal RegData corpus includes which agency and department authored each individual part of the Code of Federal Regulations, as well as the restriction and word count for each part.
 
 ## The Corpus Driver Interface
 
-Each corpus should contain a python module named `driver.py`. This driver serves two important functions. First it specifies how the corpus should be indexed. An index is one or more values that, taken together, uniquely identify each document in the corpus. An index can be as simple as an id number, or it can be more descriptive. For the CFR corpus, each document, representing a single subdivision called a part, is represented by three pieces of metadata: the year of the CFR edition that contains it, the title that contains it, and the part number.
+A corpus must contain a python module named `driver.py` in the root directory. This driver serves two important functions: indexing corpus documents, and streaming the text of those documents to other scripts.
 
-The names of the components of the index are stored in a module-level constant named `INDEX`, and are always a tuple, even when the index only has one component. Thus, for the CFR, `INDEX = ('year', 'title', 'part')`, and for a simple corpus using a document numbering system, `INDEX = ('id',)`.
+**Indexing** is assigning one or more values to each document that, when taken together, uniquely identify each document in the corpus. An index can be as simple as an identification number, or it can be more descriptive. For the Code of Federal Regulations corpus, each document, representing a single subdivision (called a 'part'), is represented by three pieces of metadata: the year of the CFR edition that contains it, the title that contains it, and the part number.
 
-The second important feature of the driver is that it provides a function named `stream` The `stream` function should return an [iterable](https://docs.python.org/3/glossary.html#term-iterable)—in most cases, a [generator](https://docs.python.org/3/glossary.html#term-generator)—that emits the index value (or values) and text of each document in the corpus. Thus, the first item emitted by `driver.stream()` in the CFR corpus might be `(1975, 1, 1), "Text of the 1975 CFR, Title 1, Part 1"`.
+The names of the components of the index are stored in a module-level constant named `INDEX`, located in `driver.py`. `INDEX` is always a tuple, even when the index only has one component. Thus, for the CFR, `INDEX = ('year', 'title', 'part')`, and for a simple corpus using a document numbering system, `INDEX = ('id',)`.
```

Comment:
Daniel, Can we update this section to talk about the corpus driver setup, and possibly the built-in drivers?